static/data/team.yml

```diff
@@ -146,6 +132,20 @@
 
 - name: Past Collaborators
   people:
+    - name: Ayoub El Khattabi
```

Comment:
We should move also: https://github.com/orgs/fastify/teams/emeritus

- Dustin Deus
- Salman Mitha
- Tommaso Allevi

Reply:
Done